data/base-spells.yaml

```diff
@@ -1167,7 +1167,12 @@ spell_data:
     skill: Utility
     abbrev: INNOCENCE
     mana: 5
-    recast: -1
+    recast: 0
+    before:
+    - message: release innocence
+    matches:
```

Comment:
❓ Does `matches` need to be indented one more level? It looks like it's part of the spell definition when it should be part of the `before`.